deploy/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fluent/fluentd:v1.6.3-debian-1.0 AS builder
+FROM fluent/fluentd:v1.8.1-debian-1.0 AS builder
```

Comment:
could we do this as a separate PR (merged before this one) to isolate the change?

Reply:
sure